e2e/cypress/integration/ui/settings/02_team_mgmt.spec.ts

```diff
@@ -17,27 +17,29 @@ describe('team management', () => {
       const admin = JSON.parse(<string>localStorage.getItem('chef-automate-user'));
       cy.cleanupV2IAMObjectsByIDPrefixes(cypressPrefix, ['projects']);
```

Comment:
Now that we're also creating policies when we create a project, let's make sure to also clean up policies in any cypress spec that creates a project.

Reply:
Good catch!